tests/specification.dylan

```diff
@@ -17,8 +17,25 @@ define interface-specification-suite testworks-interface-specification-test-suit
   open instantiable class <test-runner> (<object>);
 end;
 
+define class <expected-to-fail-class> (<object>) end;
+define variable *expected-to-fail-variable* = #t;
+define constant $expected-to-fail-constant = #"etfc";
+define function expected-to-fail-function () end;
+
+define interface-specification-suite testworks-expected-to-fail-specification-suite ()
+  variable *expected-to-fail-variable* :: <integer>
+    expected-to-fail-reason: "should be boolean";
```

Comment:
I think these look a little funny without a comma

Reply:
I do too, but adding that comma causes a HUGE increase in complexity of the macros.  Unless I'm mistaken it will force me to have one matcher for `variable v :: type;` and one matcher for `variable v :: type, expected-to-fail-reason: "r";`, plus an auxiliary macro for each spec type, to avoid too much code duplication.  Maybe it's worth it...what do you think?

Reply:
You should be able to do it without any duplication using `#rest` pattern rules, e.g. `, #rest ?test-options:*`

Reply:
Perfect, thanks!